lua/dap/utils.lua

```diff
@@ -34,25 +34,48 @@ end
 
 --- Show a prompt to select a process pid
 function M.pick_process()
-  local output = vim.fn.system({'ps', 'a'})
+  local is_windows = vim.fn.has('win32')
+  local separator = is_windows and ',' or ' \\+'
+  local command = is_windows and {'tasklist', '/nh', '/fo', 'csv'} or {'ps', 'ah'}
```

Comment:
What was the reason to add `h` to `ps a` ?

Reply:
This option disables header line in a _ps_ output, so this check can be removed:
```lua
    if pid and pid ~= 'PID' then
```